samples/boards/infineon/cy8ckit_062s2_ai_dual_dual_core/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
```

Comment:
we have a multi core sample already, why do we need another one that is board specific? Why can't you instead get yours working with the existing one https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/sysbuild/hello_world

Reply:
That sample does the same for various different types of hardware, your hardware should be updated to work the same and use the same base sample